db/migrate/20150909015851_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,54 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration
```

Comment:
this migration is duplicating `db/migrate/20150909014453_devise_token_auth_create_users.rb`